can-bus/json/BMS/BMS_rx.json

```
@@ -3,6 +3,7 @@
         "DCM_Vitals",
         "FSM_Vitals",
         "PDM_Vitals",
-        "DIM_Vitals"
+        "DIM_Vitals",
+        "Debug_CanModes"
     ]
 }
```

nit: missing blank line
```suggestion
}

```